plasmapy/dispersion/tests/test_api.py

```diff
@@ -0,0 +1,85 @@
+import astropy.units as u
+from astropy import constants
+import pytest
+
+@pytest.mark.xfail(strict=True)
+def test_dispersion_api():
+    """
+    This is a broad overview of the initial ideas we had on the sample API for the dispersion solver
+    as per https://github.com/PlasmaPy/plasmapy-meeting-notes/blob/master/2019-09-24-design-meeting.md
+
+    It is marked as xfail - ideally a later PR will cause this 
+    """
+
+    p = Particle('p')
+
+    # class Species(Particle):
```

Comment:
I don't think `Species` should subclass `Particle`, but should take a `Particle` object as one of it's `__init__` parameters. (ie. a single `Particle` can't have a temperature, but a `Species` that is comprised of lots of the same type of particle can)